### PR TITLE
Feat/clean product page

### PR DESCRIPTION
**Pull request by julienbourdeau**

### CODE REVIEW

File: templates/catalog/_partials/product-customizations.tpl [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            {if $field.is_customized}
+              <img src="{$field.image.small.url}">
+              <a class="remove-image" href="{$field.remove_image_url}" rel="nofollow">{l s='Remove Image'}</a>
+            {/if}
+            <input {if $field.required} required {/if} type="file" name="{$field.input_name}">
+          {/if}
+        </li>
+      {/foreach}
+    </ul>
+    <button name="submitCustomizedDatas">{l s='Save Customization'}</button>
```
Comment (mickaelandrieu) [inline templates/catalog/_partials/product-customizations.tpl]: should be `submitCustomizedData`

Comment (julienbourdeau) [inline templates/catalog/_partials/product-customizations.tpl]: Definitely! I'll update this

File: templates/catalog/product.tpl [hunk 2]
```diff
@@ -301,32 +301,11 @@
...
-                        {/if}
-                      </li>
-                    {/foreach}
-                  </ul>
-                  <button name="submitCustomizedDatas">{l s='Save Customization'}</button>
-                </form>
-              </section>
-            {/if}
-          {/block}
+          {if $product.is_customizable}
```
Comment (mickaelandrieu) [inline templates/catalog/product.tpl]: don't forget to refactor this part.
Also, what happens if we want to display a message when the product is not customizable ?

**Comment by mickaelandrieu** [comment]
thank you @julienbourdeau

**Pull request merged**